libs/net/models/Settings/ReportSectionsSettingsModel.cs

```diff
@@ -7,6 +7,7 @@ public class ReportSectionsSettingsModel
 {
     #region Properties
     public bool UsePageBreaks { get; set; }
+    public bool HideEmpty { get; set; }
```

Comment:
I don't think this is required.  The `hideEmpty` checkbox on the top form merely is a way to quickly turn on or off the `section.hideEmpty` checkboxes.

Reply:
I really don't know. I just added to be able to compile (and test) the template I had (I had errors on preview on these options). So I left to ask about it today.
If the template doesn't require:
IncludeStory 
ShowContent 
HideEmpty 
I'll just revert this files.
@kylgarmor any thoughts on this ?

Reply:
Each section has an option for `hideEmpty` that controls how the template behaves.  But the overall sections settings doesn't require it.  The UI form has a checkbox there to flip the checkbox on all the sections.

Reply:
Every time I tried to preview the report without this fields, I have this error:
![image](https://github.com/bcgov/tno/assets/26801490/1630fcc4-0d53-4377-9a69-0a37da560116)
It was just a workaround to make sure my real changes would work.
I can (and probably will) remove all this additional fields, but I'll need some clarification, why I'm having these errors.

Reply:
I was only using an old template to test it.
Unnecessary fields removed ... good to go !!!